documentation/DataLoader.md

```diff
@@ -0,0 +1,66 @@
+# CouchDB Data Loader
+
+(`scripts/deleteAndLoadSnapsets.sh`)
+
+This script is used to setup CouchDB database and is executed as a Kubernetes batch Job every time new version of the
+universal image is deployed to the cluster (also when cluster is initially created).
+
+It does following:
+
+- Converts the preferences in universal into `snapset` Prefs Safes and GPII Keys,
+- Optionally deletes existing database,
+- Creates a CouchDB database if none exits,
+- Updates the database with respect to its `design/views` document, as required,
+- Loads the latest snapsets created into the database.
+
+## Environment Variables
+
+- `COUCHDB_URL`: URL of the CouchDB database. (required)
```

Comment:
Can we add prefixes to these (probably GPII_) to reduce chances of conflicts with other uses of the environment (and also as a hint/courtesy to anyone looking in the environment wondering how they got there)

Reply:
Yes, that sounds like a good idea (although the chance of conflict is pretty much non-existent - data loader runs in its own container and there's nothing else expected to be running).